trillian/ctfe/sth.go

```diff
@@ -30,7 +30,7 @@ import (
 type contextKey string
 
 // remoteQuotaCtxKey is the key used to attach a Trillian quota user to
-// context.Context passed in to STH getters.
+// context.Context passed into STH getters.
```

Comment:
this correction pattern has been fairly controversial, but I believe this change is correct